nets/z2_color_lstm.py

```diff
@@ -49,7 +49,7 @@ def forward(self, x, metadata):
         # lstm1
         x = x.view(-1, 2, 256)
         x = self.lstm1(x)[0]
-        x = x.view(-1, 2 * 16)
+        x = x.contiguous().view(-1, 2 * 16)
```

Comment:
Can you explain what this does? Does it store the Tensor in one contiguous block of memory or what? Not quite sure and couldn't understand it from the PyTorch docs.